pupil-spa/docker-start.sh

```diff
@@ -4,7 +4,7 @@
 /usr/sbin/sshd
 
 # Always re-generate the config based on the env variables
-sh gen_config.sh && mv config.json /usr/share/nginx/html/public
+./gen_config.sh && mv config.json /usr/share/nginx/html/assets
```

Comment:
is this exposed under the same route in the angular app?  if not, the app will need updating to reflect it

Reply:
The app has already been updated to use this path in the merged Angular upgrade PR.